src/c++/vdo/base/int-map.h

```diff
@@ -24,9 +24,9 @@
 struct int_map;
 
 int __must_check
-vdo_make_int_map(size_t initial_capacity, unsigned int initial_load, struct int_map **map_ptr);
+vdo_int_map_create(size_t initial_capacity, unsigned int initial_load, struct int_map **map_ptr);
 
```

Comment:
We skipped reformatting int-map, so could we do it in this series? In the second change seems reasonable.

I think just the signatures for vdo_int_map_create() and vdo_int_map_put() need to get wrapped, in both the header and the .c file.

.... If you reorder the patches to remove the initial_load parameter before the name changes, you shouldn't have to reflow the int_map_create() lines. That'd probably be gruesome to resolve though, maybe just fix the int_map_create() wrapping knowing you'll be shortening the lines lafter.

Reply:
int-map.c and int-map.h should be formatted now.